cmd/snap/cmd_debug_validate_seed.go

```diff
@@ -0,0 +1,55 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/snapcore/snapd/snap"
+)
+
+type cmdValidateSeed struct {
+	Positionals struct {
+		SeedYamlPath string `positional-arg-name:"<seed-yaml-path>"`
+	} `positional-args:"true"`
+}
+
+func init() {
+	cmd := addDebugCommand("validate-seed",
```

Comment:
or `validate-seed-file` ?

Reply:
Not sure, feel a bit long but I don't really mind strongly either way. Its a debug command after all :)